sys/crypto/dragon.c

```diff
@@ -0,0 +1,321 @@
+/*
+ * Copyright (C) 2015 Jan Wagner <mail@jwagner.eu>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_crypto
+ * @{
+ *
+ * @file        dragon.c
```

Comment:
Remove the parameter from the `@file` Doxygen command (but leave the `@file`), Doxygen will figure out the filename by itself. This comment applies to all files in the PR.